### ISSUE TITLE
Fix tests that are disabled in Edge

### ISSUE THREAD

**Issue by zcorpan** [w3c/web-platform-tests#10655]
@thejohnjansen can you provide a list of web-platform-tests that are disabled or marked as flaky or slow in Edge?

**Comment by thejohnjansen** [comment]
Sorry about the delay. I'm working to verify that what I think is correct :-)

**Comment by thejohnjansen** [comment]
Here is a CSV of the tests we have disabled currently. Of course, we run these on a regular cadence to see if they have become stable, and we add to the list and new tests become unstable.

[NotRunFiles.xlsx](https://github.com/w3c/web-platform-tests/files/1984479/NotRunFiles.xlsx)

**Comment by thejohnjansen** [comment]
GitHub does not support .CSV, so it's actually an .XLSX, but it really is just a csv.

**Comment by zcorpan** [comment]
It looks like the file got mangled. :-( Can you attach it as a .txt, or create a [gist](https://gist.github.com/), or put it on OneDrive or something?

**Comment by zcorpan** [comment]
Oh I see, it's actually .xlsx, not just a file rename. OK, that works. Thanks!

**Comment by zcorpan** [comment]
@thejohnjansen my priority right now is to process the tests that are common for 2 or more browsers. Can someone at Microsoft go through and fix tests (if they need fixing) the tests from my report that are disabled in only Edge?

**Comment by zcorpan** [comment]
https://bocoup.github.io/wpt-disabled-tests-report/#filter=edge shows just the Edge rows.

**Comment by inikulin** [comment]
@zcorpan Hi Simon, is this data available in a machine-readable format somewhere?

**Comment by zcorpan** [comment]
Yes, the report links to it: https://bocoup.github.io/wpt-disabled-tests-report/common.json

**Comment by zcorpan** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18774235/

**Comment by zcorpan** [comment]
I changed the priority for this issue since EdgeHTML is no longer being developed. Though the tests might still need fixing.

**Comment by zcorpan** [comment]
The file linked above is no longer available. I had a local copy. I converted it back to csv and uploaded as a gist here

https://gist.github.com/zcorpan/b1a3272da3cca52e208b643416d10967

**Comment by zcorpan** [comment]
The file is available again. Weird :)
